netkan/netkan/indexer.py

```diff
@@ -137,10 +137,13 @@ def _process_ckan(self):
         try:
             status = ModStatus.get(self.ModIdentifier)
             attrs = self.status_attrs()
+            if not self.Success and status.last_error != attrs.last_error:
+                logging.error('New inflation error for %s: %s',
+                              self.ModIdentifier, attrs.last_error)
```

Comment:
This is potentially spammy, because error messages can include the name of the temporary download file (e.g. invalid ZIP files), and that one changes every run.
Currently this is only `TrimIndicator` though.

Reply:
Yeah, I am leaning towards removing the filename from that error. Can you think of a reason to keep it?

Reply:
Well, it helps finding the file for investigation purposes, but if you debug-log the file name, it should be enough.

Reply:
Fixed in KSP-CKAN/CKAN#2903.